File: DesignPatternsExercises/java/ChainOfResponsibilityDesignPattern/Exercise/ProductionSupport.java
public class ProductionSupport extends __________(10)__________ {
	@Override
	public void handleIssue(IssueType issueType) {
		if(issueType == IssueType.Production) {
			System.out.println("Production support team will contact you shortly");
		} else {
			__________(11)__________(issueType);
		}
	}
}

File: DesignPatternsExercises/ruby/chain_of_responsibility_design_pattern/exercise/production_support.rb
require_relative 'support_chain'

class ProductionSupport < SupportChain
  def handle_issue(issue_type)
    if issue_type == :production
      puts 'Production support team will contact you shortly'
    else
      @next_support_link.__________(3)__________(issue_type)
    end
  end
end
